lib/Database.js

```diff
@@ -149,6 +152,20 @@ var Database = module.exports = Class.extend({
   },
 
   /**
+   * Create the order by query segment
+   */
+  getOrderBy: function (orders) {
+    if (typeof orders == 'string' || orders instanceof String) return ' ORDER BY ' + orders;
+    if (!orders || (orders && orders.length === 0)) return '';
+    var query = ' ORDER BY';
+    for (var i in orders) {
+      var order = orders[i];
+      query += ' ' + order[0] + (order[1] ? ' ' + order[1].toUpperCase(): '') + ((orders.length - 1) != i ? ',': '');
```

Comment:
Could you please ensure that this isn't open to SQL injection? You could add a "sanitize" method like the "quote" method for order[0], and for order[1], maybe use:
```
var direction = (order[1] || '').replace(/^.*?(asc|desc)?.*?$/, '$1');
```